druid-shell/src/platform/x11/window.rs

```diff
@@ -763,15 +844,55 @@ impl Window {
     }
 
     /// Set whether the window should be resizable
-    fn resizable(&self, _resizable: bool) {
-        warn!("Window::resizeable is currently unimplemented for X11 platforms.");
+    fn resizable(&self, resizable: bool) {
+        let conn = self.app.connection().as_ref();
+        log_x11!(size_hints(resizable, self.size().size_px(), self.min_size)
+            .set_normal_hints(conn, self.id)
+            .context("set normal hints"));
     }
 
     /// Set whether the window should show titlebar
     fn show_titlebar(&self, _show_titlebar: bool) {
         warn!("Window::show_titlebar is currently unimplemented for X11 platforms.");
     }
 
+    fn get_position(&self) -> Point {
+        fn _get_position(window: &Window) -> Result<Point, Error> {
+            let conn = window.app.connection();
+            let scale = window.scale.get();
+            let geom = conn.get_geometry(window.id)?.reply()?;
+            let cord = conn
+                .translate_coordinates(window.id, geom.root, geom.x, geom.y)?
+                .reply()?;
```

Comment:
Shouldn't the above be `translate_coordinates(window.id, geom.root, 0, 0)`? You want to know which coordinates the point (0,0) in your window has in the root window. As written, this function figures out the position of the window inside of its parent window (which will be the size of the top/left window decorations in a reparenting WM) and then translates this to the root window.

Reply:
Thanks :)